src/components/SingleReview.jsx

```diff
@@ -112,16 +149,36 @@ const SingleReview = () => {
             ) : null}
           </div>
         </div>
-        <div className="">
+
+        <div className="view_comments">
           {singleReview.comment_count === 0 ? (
-            <Typography> No Comments... Yet!</Typography>
+            <div className="add_comment">
+              <Typography> No Comments... Yet!</Typography>
+              <TextField
+                sx={{ m: 1 }}
+                id="outlined-adornment-amount"
+                label="Add Comment"
+                onChange={(event) => {
+                  setUserComment(event.target.value);
```

Comment:
This is almost a controlled component - the value inside the textField needs to come from the state. 
You're setting the state on change, but not using the value here at the moment.